lib/matplotlib/axes/_axes.py

```diff
@@ -1091,55 +1091,72 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
                   linelengths=1, linewidths=None, colors=None,
                   linestyles='solid', **kwargs):
         """
-        Plot identical parallel lines at specific positions.
+        Plot identical parallel lines at the given positions.
 
-        Plot parallel lines at the given positions.  positions should be a 1D
-        or 2D array-like object, with each row corresponding to a row or column
-        of lines.
+        *positions* should be a 1D or 2D array-like object, with each row
```

Comment:
See discussion starting here: https://github.com/matplotlib/matplotlib/pull/8172#discussion_r103843079
ref http://matplotlib.org/devdocs/devel/documenting_mpl.html#formatting
ref https://docs.python.org/devguide/documenting.html#id3

Let's keep the asterisks unless there is agreement to switch to double backquotes (which I would not favor).  If we go with such a switch the documenting matplotlib guide should be updated first.